packages/devtools_app/test/performance_model_test.dart

```diff
@@ -90,7 +91,12 @@ void main() {
               'vsyncOverhead': 1000
             },
           ],
-          PerformanceData.cpuProfileKey: goldenCpuProfileDataJson,
+          PerformanceData.cpuProfileKey:
+              (await CpuProfileData.generateFromCpuSamples(
```

Comment:
Did `dart format` do this?

Reply:
yeah,  I think so. I have vscode auto format every time i save my files 😅

Reply:
without the await this looks much cleaner now :D